src/main/java/com/softserveinc/ita/pageobjects/modals/AddingFormModal.java

```diff
@@ -3,22 +3,45 @@
 import com.softserveinc.ita.models.AddingFormFields;
 import io.qameta.allure.Step;
 
-import static com.codeborne.selenide.Condition.appear;
-import static com.codeborne.selenide.Condition.visible;
+import java.util.Objects;
+
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$$x;
 import static com.codeborne.selenide.Selenide.$x;
 import static java.time.Duration.ofSeconds;
 
 public class AddingFormModal {
 
     @Step("Adding form modal: Set {value} for {field}")
     public AddingFormModal setValueFor(AddingFormFields field, String value) {
-        field
+        var fieldElement = field
                 .getName()
-                .should(appear, ofSeconds(5))
-                .sendKeys(value);
+                .should(appear, ofSeconds(5));
+
+        if (Objects.requireNonNull(fieldElement
```

Comment:
static import of "requireNonNull"

extract into variable "fieldElement.getAttribute("class")"